framework_tvb/tvb/core/services/burst_service.py

```diff
@@ -315,7 +316,10 @@ def prepare_metrics_operation(operation):
         range_values = operation.range_values
         metric_algo = dao.get_algorithm_by_module(MEASURE_METRICS_MODULE, MEASURE_METRICS_CLASS)
 
-        metric_operation = Operation(operation.fk_launched_by, operation.fk_launched_in, metric_algo.id, '',
+        adapter = ABCAdapter.build_adapter(metric_algo)
+        view_model = adapter.load_view_model(operation)
```

Comment:
Is this `ViewModel` properly stored in an H5 ?

Reply:
I don't think this fix is correct, here you would load the view_model of the simulation, not the view_model of the metrics `TimeseriesMetricsAdapterModel`. I added a commit to store the metrics view_model on HPC and then stage it out and update the db accordingly.